### PR TITLE
LITE-19396: Add intro and summary dialogs on EAAS bootstrap

### PR DESCRIPTION
**Pull request by jpdolz**

### CODE REVIEW

File: tests/plugins/project/test_extension_helpers.py [hunk 1]
```diff
@@ -93,7 +95,7 @@ def test_bootstrap_extension_project(
     captured = capsys.readouterr()
     assert 'project_dir' in captured.out
     assert mocked_cookiecutter.call_count == 1
-    assert mocked_dialogus.call_count == 6
+    assert mocked_dialogus.call_count == 8
 
 
 def test_bootstrap_direxists_error(fs, mocker, config_mocker):
```
Comment (gab832) [inline tests/plugins/project/test_extension_helpers.py:101]: Do you mean _dir_exists_error?

**Review by gab832 — APPROVED** [review]
It looks good.

File: connect/cli/plugins/project/utils.py [hunk 2]
```diff
@@ -35,6 +35,126 @@ def _run_hook_from_repo_dir(
     pass
 
 
+def eaas_introduction(config, idx, total):
+    intro = [
+        {
+            'name': 'introduction',
+            'type': 'input',
+            'message': 'Welcome to the Extension as a Service boilerplate utility.\n'
```
Comment (marcserrat) [inline connect/cli/plugins/project/utils.py:43]: Welcome to the Connect Extension Project Bootstrap utility

File: connect/cli/plugins/project/utils.py [hunk 3]
```diff
@@ -35,6 +35,126 @@ def _run_hook_from_repo_dir(
...
+            'You will drive through different screen dialogs where you provide required\n'
```
Comment (marcserrat) [inline connect/cli/plugins/project/utils.py:44]: Using this utility multiple data will be collected in order to create a project skeleton for your Connect Extension. This skeleton will contain all the needed things to start writing your code.

File: connect/cli/plugins/project/utils.py [hunk 4]
```diff
@@ -35,6 +35,126 @@ def _run_hook_from_repo_dir(
...
+    return answers
+
+
+def eaas_summary(answers, idx, total):
+    summary_list = _prepare_summary(answers)
+    summary = [
+        {
+            'name': 'summary',
+            'type': 'input',
+            'message': 'The selected values for the extension project have been:\n\n'
```
Comment (marcserrat) [inline connect/cli/plugins/project/utils.py:61]: The provided values for the project are:

**Review by marcserrat — CHANGES_REQUESTED** [review]

**Pull request merged**